### DISCUSSION TITLE
Lidl Button

### DISCUSSION THREAD

**Discussion by Kasperfcb** [Koenkk/zigbee2mqtt#12578] [Devices]
hi I have a Lidl HG08164 button but can not get it to show battery some that can help

**Comment by rolo64416** [comment]
I have found that some battery operated sensors wil show battery information after is has been on for some time (several hours).

**Comment by JimGermany** [comment]
Same problem. Button is working (now for two days) but Batterie = N/A

Zigbee Modell: TS004F
Zigbee Hersteller: _TZ3000_rco1yzb1

With:
- Sonoff Zigbee 3.0 USB Dongle Plus
- Zigbee2MQTT: 1.25.2-1
- Home Assistant: 2022.3.8
state_class: measurement
action: 
battery: null
linkquality: 102
unit_of_measurement: %
device_class: battery
friendly_name: Lidl_SB_1 battery

**Comment by peteh** [comment]
Same for me, the battery status is never shown. I have two buttons - one new for a week now and the other one for 2 months.

**Comment by HeadCrash66** [comment]
Hi,

I'm new to Zigbee an Z2M ... I got a Lidl HG08164 and running Z2M under Linux with ELU013 (Popp) USB adapter.

Pairing works fine. I got linkquality, which changes if I click the button and move around. So I guess communication works.

*But* i do not get any action. I keeps "N/A".

Any one any idea what i may do wrong?

**Reply by peteh** [reply]
It's not related to this issue but if you have the lidl button, it has two modes. Press the button quickly 3 times in a row (led should light up to confirm that mode is switched). This will switch the mode between scene assignment (not sure what this actually is, never used it) and the actual button mode.

**Comment by evlo** [comment]
Also no battery reported for me, maybe, so maybe create an issue and someone more smart will fix it in code? It is very minor annoyance, mostly just icon on the dashboard.

If you do trick mentioned here https://github.com/Koenkk/zigbee2mqtt/discussions/10593#discussioncomment-2059695 it will show the battery

I have created PR to add it to docu https://github.com/Koenkk/zigbee2mqtt.io/pull/1547

**Reply by Pwnyrider** [reply]
Hi,

I got the same issue - no battery reported. I tried the trick mentioned in  https://github.com/Koenkk/zigbee2mqtt/discussions/10593#discussioncomment-2059695 but still could not convince the device to show the battery state.
I tried it several times with no success. 
Here is what I did: 
* remove the device from z2m
* start device discovery in z2m
* start pairing mode on device
* wait for z2m log message "successful interview"
* push the pairing button once 

Am I doing something wrong?

**Comment by genrihb** [comment]
hi! anybody can help me?
![Снимок экрана от 2022-11-09 12-45-45](https://user-images.githubusercontent.com/105509954/200811811-4245086f-29e5-4ab5-b308-64c165b77966.png)


![Снимок экрана от 2022-11-09 12-46-16](https://user-images.githubusercontent.com/105509954/200812490-963f6bb0-84a1-48b7-b3f1-1054f511337c.png)

**Comment by JimGermany** [comment]
With the last Update 1.28.2-1 everything is OK for me and now I can see the battery level.

Home Assistant 2022.10.5
Supervisor 2022.10.2
Operating System 9.3
